src/codeu/chat/util/store/Store.java

```diff
@@ -74,6 +74,12 @@ public VALUE first(KEY key) {
   }
 
   @Override
+  public VALUE last(KEY key) {
```

Comment:
Is this method actually used anywhere? Regardless, because maps can only have one key, this seems like it would be the same as `first()`?